### PR TITLE
js-dom task

### PR DESCRIPTION
**Pull request by andreymashko7**
### task_js_dom

[Demo](https://andreymashko7.github.io/side-menu/) |
[Code base](https://github.com/andreymashko7/side-menu)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by andreymashko7** [comment]
Please rewiew

File: submissions/AndriyM/js-dom/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                </div>`
+        )
+        .join("")
+}
+
+const cardsMarcup = createCardsMarcup(cards)
+rootElement.innerHTML = cardsMarcup
+const cardRefs = document.querySelectorAll(".card__item")
+
+sideBarContainer.addEventListener("click", onsideBarElementClick)
```
Comment (madmaxWMFU) [inline submissions/AndriyM/js-dom/index.js:23]: Format your code. It is helpful to use [Prettier](https://prettier.io/).

File: submissions/AndriyM/js-dom/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+function onsideBarElementClick({ target }) {
+    links.forEach((link) => link.classList.remove("active"))
+    target.classList.add("active")
+    let currentActivetLink = target.dataset.id
```
Comment (madmaxWMFU) [inline submissions/AndriyM/js-dom/index.js:28]: Must be `const`

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good work! See some comments below.

**Comment by andreymashko7** [comment]
All ready, please check

**Comment by madmaxWMFU** [comment]
You always forget to put `;`.
Use [prettier](https://prettier.io/) to pretty format your code.

**Review by madmaxWMFU — APPROVED** [review]
Great, I approve of your work. Good luck with future tasks!

**Pull request merged**